### PR TITLE
bt-78: Create modal component

### LINKED ISSUE #78: Create Modal component
Image

https://mui.com/material-ui/react-modal/

### PR DESCRIPTION
**Pull request by Simpsonkk**

### CODE REVIEW

**Comment by Simpsonkk** [comment]
image
image
image
image

File: frontend/src/bundles/common/components/modal/modal.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { useCallback, useState } from 'react';
+
+import { Button, Typography } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+    children: React.ReactNode;
+    headerLabel: string;
+    openButtonLabel: string;
+    className?: string;
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/common/components/modal/modal.tsx]: We need to pass isOpen, onClose also

File: frontend/src/bundles/common/components/modal/modal.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                                        className={styles.closeIcon}
+                                    />
+                                }
+                            />
+                        </div>
+                    </div>
+                    <Divider />
+                    <div className={className}>{children}</div>
+                </div>
+            </MUIModal>
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/common/components/modal/modal.tsx]: You need to leave only this part of component, just create a wrapper for modal component we pass isOpen, onClose... etc props to this component

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/modal/modal.tsx (hunk 1 above)
Comment (Simpsonkk) [reply frontend/src/bundles/common/components/modal/modal.tsx]: done

File: frontend/src/bundles/common/components/modal/modal.tsx (hunk 2 above)
Comment (Simpsonkk) [reply frontend/src/bundles/common/components/modal/modal.tsx]: unfortunately, I always try to complicate tasks. Done.

Comment (MaksymGavatuk) [reply frontend/src/bundles/common/components/modal/modal.tsx]: I understand because in my first steps I was the same )

**Review by MaksymGavatuk — APPROVED** [review]

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]
Please also export this component from components.ts file

**Comment by Simpsonkk** [comment]
> Please also export this component from components.ts file

@anton-otroshchenko 
Done

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**